src/__tests__/config.test.ts

```diff
@@ -39,7 +40,6 @@ describe("Configuration", () => {
       "content-length",
       "range"
     ],
-    sendEnvironmentMetadata: true,
```

Comment:
Why is this config option removed as a default everywhere?

Reply:
I was trying to ensure through typing that these mappings were consistent (that they mapped to config options that actually existed) and in the process I removed the "inconsistencies", the config options that were no longer declared in the `AppsignalOptions` type but still had a declared mapping. For example, there was a mapping for `instrumentPg` -- but `instrumentPg` is no longer a config option, so I removed the mapping.

For `sendEnvironmentMetadata`, I removed it because it was already undeclared on `AppsignalOptions` (meaning TS users already couldn't use it) and it is marked as deprecated in [our configuration options documentation page](https://docs.appsignal.com/nodejs/2.x/configuration/options.html#option-sendenvironmentmetadata), and I could not find any code in the integration that used it.

But looking at it now, it seems like [the agent does need the environment variable to be explicitly passed and set to true](https://github.com/appsignal/appsignal-agent/blob/99ae7938e880081ecfc252e7d3b87216143ba3b1/common/src/config.rs#L272-L275), or else [it won't schedule the environment metadata to be sent](https://github.com/appsignal/appsignal-agent/blob/99ae7938e880081ecfc252e7d3b87216143ba3b1/agent/src/worker/mod.rs#L105-L108).

So, what do we do? Options that come to mind:
- Un-deprecate the config option in the docs and add it back to the `AppsignalOptions` type
- Have the Node.js integration always pass it as true
- Have the agent default it to true

Reply:
> it is marked as deprecated in [our configuration options documentation page](https://docs.appsignal.com/nodejs/2.x/configuration/options.html#option-sendenvironmentmetadata), and I could not find any code in the integration that used it.

@unflxw It shouldn't be deprecated. I don't know why it says that. It enables a thing in the agent and should be available as an option to turn it off.

> Un-deprecate the config option in the docs and add it back to the AppsignalOptions type

Yes.